psamm/datasource/sbml.py

```diff
@@ -907,6 +917,13 @@ def write_model(self, file, model):
             lower_str, upper_str = self._get_flux_bounds(
                 eq_id, model, flux_limits, equation)
 
+            params[upper_str] = self._transform_param(upper_str)
+            params[lower_str] = self._transform_param(lower_str)
```

Comment:
You could prepend the `'P_' + ` here and then you don't have to do that in the three places below.